src/bases.jl

```diff
@@ -48,7 +48,7 @@ Abstract type that represents a basis of vector space of type `VST` on a manifol
 a subset of it.
 
 The type parameter `𝔽` denotes the [`AbstractNumbers`](@ref) that will be used
-for the vectors elements.
+as coefficients in linear combinations of vectors.
```

Comment:
```suggestion
as coefficients in linear combinations of the basis vectors.
```
maybe a bit more precise this way? (he other places accordingly of course)